src/realm/realmReducer.js

```diff
@@ -30,6 +35,34 @@ const initialState = {
 const convertRealmEmoji = (data): RealmEmojiById =>
   objectFromEntries(Object.keys(data).map(id => [id, { ...data[id], code: id.toString() }]));
 
+/**
+ * The video chat provider configured by the server. If the server has
+ * configured a provider that is not currently supported by the mobile app, this
+ * function should return null.
+ *
+ * To implement additional providers, see the `VideoChatProvider` type in
+ * `src/reduxTypes.js`.
+ */
+function getVideoChatProvider({
+  availableProviders,
+  jitsiServerUrl,
+  providerId,
+}: {
+  availableProviders: AvailableVideoChatProviders,
+  jitsiServerUrl?: string,
+  providerId: number,
+}): VideoChatProvider | null {
+  if (
+    availableProviders.jitsi_meet &&
+    availableProviders.jitsi_meet.id === providerId &&
+    jitsiServerUrl != undefined
```

Comment:
If you run our test suites with `tools/test`, I think you'll see some lint errors here. You can fix some of them automatically with `--fix`; see `tools/test --help`.

(This should run automatically as CI when you send a PR, but our CI provider has been flaking on us lately; we should probably switch. That's #4174. Fortunately it's easy to run locally.)